cove/cove_360/templates/cove_360/cookie_consent.html

```diff
@@ -1,7 +1,9 @@
 <!-- Matomo -->
 <script type="text/javascript">
   var _paq = window._paq = window._paq || [];
-  var siteID = 19;
+  {% if piwik.site_id %}
+  var siteID = {{piwik.site_id}};
```

Comment:
Could we update these variables (`_paq` and `siteID`) to use `const` instead of `var`?

Reply:
I do agree ...however this starts to get into the realm of refactoring, if I do this one then the others become inconsistent unless I address them all, which is currently out of scope.

Reply:
Agreed, one for the future maybe.